### PR TITLE
You can know assign a project leader to a project

### PR DESCRIPTION
**Pull request by mads10m**

### CODE REVIEW

**Comment by PhiltasticPancakes** [comment]
Merge master into this branch and resolve merge conflicts

**Review by PhiltasticPancakes — CHANGES_REQUESTED** [review]
Merge master into this branch and resolve merge conflicts

File: use_cases/assign_project_leader.feature [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
+	And there exists a project with name "P1"
+	When a worker assign the project "P1" a project leader
+	Then the project "P1" has a project leader with id "ASDF"
+
+
+Scenario: Change project leader
+	Given that a worker is logged in with id "ASDF"
+	And there is a project with name "P1"
+	And the project "P1" has a project leader
+	When a worker change the project leader on project "P1"
```
Comment (PhiltasticPancakes) [inline use_cases/assign_project_leader.feature]: changes

File: use_cases/assign_project_leader.feature [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
-#	Then the project "p1" has a new project leader
\ No newline at end of file
+Feature: Assign project leader
+	Description: A worker assign a project leader to a project
+	Actor: Worker
+
+Scenario: Assign project leader
+	Given that a worker is logged in with id "ASDF"
+	And there exists a project with name "P1"
+	When a worker assign the project "P1" a project leader
```
Comment (PhiltasticPancakes) [inline use_cases/assign_project_leader.feature]: assigns

File: src/test/java/dtu/scheduler/ProjectLeaderTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+		
+	}
+
+	@Given("there is a project with name {string}")
+	public void thereIsAProjectWithName(String name) throws ProjectAlreadyExistsException {
+		schedulingApp.addProject(new Project(name));
+	}
+
+	@When("a worker change the project leader on project {string}")
+	public void aWorkerChangeTheProjectLeaderOnProject(String projectID) {
```
Comment (PhiltasticPancakes) [inline src/test/java/dtu/scheduler/ProjectLeaderTest.java:50]: changes

**Review by PhiltasticPancakes — CHANGES_REQUESTED** [review]

**Review by mads10m — COMMENTED** [review]
Thanks

**Review by mads10m — COMMENTED** [review]
Looks good

**Review by PhiltasticPancakes — APPROVED** [review]

**Pull request merged**